dev/com.ibm.ws.messaging.open_jms20deliverydelay_fat/fat/src/com/ibm/ws/messaging/jms20/deliverydelay/fat/DelayFullTest.java

```diff
@@ -0,0 +1,584 @@
+        msg = clientServer.waitForStringInLogUsingLastOffset(
+            "Message received on mdb : TopicBindingsMessage1-ClassicApi");
+        assertNotNull("Could not find the upload message in the trace.log", msg);
+
+        restartServers(MDB_CONFIG_TOPIC_TCP);
+
+        runInServlet("testDeliveryDelayForDifferentDelaysTopicClassicApi_Tcp");
+        msg = clientServer.waitForStringInLogUsingLastOffset(
+            "Message received on mdb : TopicTCPMessage2-ClassicApi");
+        assertNotNull("Could not find the upload message in the trace.log", msg);
+        msg = clientServer.waitForStringInLogUsingLastOffset(
+            "Message received on mdb : TopicTCPMessage1-ClassicApi");
+        assertNotNull("Could not find the upload message in the trace.log", msg);
+
+        restartServers();
+    }
+
+    @Test
+    public void testPersistentMessageStoreClassicApi_B() throws Exception {
+        runInServlet("testPersistentMessageClassicApi");
+
+        restartServers();
+
+        boolean testResult = runInServlet("testPersistentMessageReceiveClassicApi");
+        assertTrue("testPersistentMessageStoreClassicApi_B failed", testResult);
+    }
+
+    @Test
+    public void testPersistentMessageStoreClassicApi_Tcp() throws Exception {
+        runInServlet("testPersistentMessageClassicApi_Tcp");
+
+        restartServers();
+
+        boolean testResult = runInServlet("testPersistentMessageReceiveClassicApi_Tcp");
+        assertTrue("testPersistentMessageStoreClassicApi_Tcp failed", testResult);
+    }
+
+    @Test
+    public void testPersistentMessageStoreTopicClassicApi_B() throws Exception {
+        runInServlet("testPersistentMessageTopicClassicApi");
+
+        restartServers();
+
+        boolean testResult = runInServlet("testPersistentMessageReceiveTopicClassicApi");
+        assertTrue("testPersistentMessageStoreTopicClassicApi_B failed", testResult);
+    }
+
+    @Test
+    public void testPersistentMessageStoreTopicClassicApi_Tcp()throws Exception {
+        runInServlet("testPersistentMessageTopicClassicApi_Tcp");
+
+        restartServers();
+
+        boolean testResult = runInServlet("testPersistentMessageReceiveTopicClassicApi_Tcp");
+        assertTrue("testPersistentMessageStoreTopicClassicApi_Tcp failed", testResult);
+    }
+
+    // regression tests for durable unshared
+
+    @Test
+    public void testCreateUnSharedDurable_B_SecOff() throws Exception {
+        boolean testResult = true;
+
+        if ( !runInServlet("testCreateUnSharedDurableConsumer_create") ) {
+            testResult = false;
+        }
+        
+        restartClient();
+
+        if ( !runInServlet("testCreateUnSharedDurableConsumer_consume") ) {
+            testResult = false;
+        }
+
+        assertTrue("testCreateSharedDurableExpiry_B_SecOff failed", testResult);
+    }
+
+    @Test
+    public void testCreateSharedDurable_B_SecOff() throws Exception {
+        boolean testResult = true;
+
+        // TODO: ???
```

Comment:
Remove '// TODO: ???`  My concern here is that the test methods from this point forward changed pattern for computing testResult.  Odd.

Reply:
Noted; the comment was removed.